internal/pkg/metrics/prometheus.go

```diff
@@ -0,0 +1,43 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/sirupsen/logrus"
+	"net/http"
+)
+
+type Collectors struct {
+	Reloaded *prometheus.CounterVec
+}
+
+func NewCollectors() Collectors {
+	reloaded := prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "reloader",
+			Name:      "reload_executed_total",
```

Comment:
I was seriously thinking what is the best name of this metrics... But, I couldn't reach good naming.